test/static-code

```diff
@@ -31,9 +31,12 @@ find_python_files() {
     find_scripts 'python3' '*.py'
 }
 
-test_flake8() {
-    command -v flake8 >/dev/null || skip 'no flake8'
-    find_python_files | xargs -r -0 flake8
+test_pycodestyle() {
+    # We run pycodestyle to cover rules not yet implemented by ruff
+    # See https://github.com/astral-sh/ruff/issues/2402
+    # and https://github.com/astral-sh/ruff/releases/tag/v0.0.269
+    command -v pycodestyle >/dev/null || skip 'no pycodestyle'
+    find_python_files | xargs -r -0 pycodestyle --select=E1,E2,E3
```

Comment:
I'm not that happy about this. Having `.flake8` had the big advantage that vim ALE and friends used the same configuration as the tests. pycodestyle does not have such a facility.

Could this work with opting into that subset of rules with flake8 instead? Then we also won't need to change the dependency *again* in all our projects, tasks container, etc. There's nothing wrong with flake8 per-se, is it?

Reply:
Note: flake8 implements the rules in question internally by using pycodestyle.  The flake8 package depends on pycodestyle (and a whole lot of other stuff), so if you have flake8 you already have pycodestyle.

We could also add a `setup.cfg` with the pycodestyle config in it...

Reply:
> We could also add a setup.cfg with the pycodestyle config in it...

That'd be great! I'm aware of the flake8 → pycodestyle dependency, I'm just worried about a more declarative config.